pipeline/azure-pipelines.yml

```diff
@@ -159,6 +159,15 @@ jobs:
       maximumCpuCount: true
       logFileVerbosity: diagnostic
 
+  - task: CopyFiles@2
+    displayName: 'Copy asmref files to: $(Build.ArtifactStagingDirectory) in the event of a build failure' 
+    inputs:
+      SourceFolder: $(Build.SourcesDirectory)\src\Integration.Vsix
+      Contents: |
+        **\AsmRef_*.txt
+      TargetFolder: $(Build.ArtifactStagingDirectory)
+    condition: failed()     # we're only interested if the build failed (in case the build failed because of a mismatch in the asmref files)
+
```

Comment:
I couldn't work out why I was getting CI build failures in a PR because of differences in the asmref outputs, so I tweaked the build pipeline to publish the asmref files in the event of a build failure.

Reply:
Very nice!